redaxo/src/core/pages/system.log.redaxo.php

```diff
@@ -85,9 +92,10 @@
 $fragment = new rex_fragment();
 $fragment->setVar('elements', $formElements, false);
 $buttons = $fragment->parse('core/form/submit.php');
+$logFileLink = '<a href="' . rex_url::currentBackendPage(['func' => 'download']) . '">' . $logFile . '</a>';
 
 $fragment = new rex_fragment();
-$fragment->setVar('title', rex_i18n::msg('syslog_title', $logFile), false);
+$fragment->setVar('title', rex_i18n::rawmsg('syslog_title', $logFileLink), false);
```

Comment:
```suggestion
$fragment->setVar('title', rex_i18n::rawMsg('syslog_title', $logFileLink), false);
```

Wobei man mit rawMsg auch immer aufpassen muss. Die Message hier enthält zum Beispiel Double quotes, die nun nicht mehr zu `&quot;` werden. Da es nicht in einem HTML-Attribut ausgegeben wird, aber vermutlich hier kein Problem.

Wir könnten aber auch nicht hier oben die Datei verlinken, sondern unten einen weiteren Button platzieren, neben dem, um die Datei im Editor zu öffnen (nur vorhanden, wenn Editor unter System ausgewählt):

<img width="882" alt="Screenshot 2020-01-28 10 30 00" src="https://user-images.githubusercontent.com/330436/73252319-d9155d80-41ba-11ea-823c-fdb260c17968.png">